examples/example_powermeter_mqtt.py

```diff
@@ -0,0 +1,193 @@
+"""
+Example of a daemon listening for values from my main power-meter and resend them on a MQTT bus.
+
+This example will not be able to run as it is - but it will homefully give you some
+ideas to how you can define DPT, and get their converted values, and even send them to MQTT
+in a tested topic-format.
+
+I have a Mosquitto MQTT Server - and running the Paho Python client.
+I use some external MQTT libraries as well to handle the MQTT Topic-creation.
+
+The data published on the MQTT bus can be fetched and stored in InfluxDB for graphing,
+or monitored by other listeners - that triggers different events.
+
+Please join XKNX on Discord (https://discord.gg/5XARFNT) and chat with JohanElmis for specific questions.
+"""
+
+from __future__ import print_function
+
+try:
+    import asyncio
+
+    from xknx import XKNX
+    from xknx.devices import Sensor
+    import sys
+    import re
+    # import time
+    import paho.mqtt.client as mqtt
+    # The following library is not included.
+    from myhouse_sensors_mqtt import SensorClientMqtt
+    from myhouse_sensors_mqtt import MetricType
+    # import pprint
+except ImportError as import_err:
+    err_list = str(import_err).split(' ')
```

Comment:
I would add a `# pylint disable=invalid-name` here. Pylint thinks it is a constant, but it obviously isn't...